assets/dev/js/editor/document/commands/base/command-disable-enable.js

```diff
@@ -0,0 +1,94 @@
+import CommandHistory from './command-history';
+import ElementsSettings from 'elementor-document/elements/commands/settings';
+
+export default class CommandDisableEnable extends CommandHistory {
+	static getName() {
+		elementorModules.ForceMethodImplementation();
```

Comment:
Probably it's not related to this PR, but according to our standards, every function name must start with a lower-case character

Reply:
![image](https://user-images.githubusercontent.com/10234691/84771215-ae993500-afe1-11ea-9d77-494110340200.png)
Yes + Location is not right. 

need to add it to maintenance and fixes.

Reply:
@matipojo, maybe add 'task'?

Reply:
The TODO is enough.  fix the TODO to `forceMethodImplementation`.